packages/cardhost/tests/integration/helpers/scope-css-test.js

```diff
@@ -0,0 +1,15 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Helper | scope-css', function(hooks) {
+  setupRenderingTest(hooks);
+
+  // Replace this with your real tests.
+  test('it renders', async function(assert) {
+    await render(hbs`{{scope-css ".some-class { font-size: 10px; }" "my-card"}}`);
```

Comment:
Just a suggestion to have the test example css have 2 styles to demonstrate that it prefixes both of them with the parent selector. I didn't realize that we had a plugin helping us until I looked a bit further into the implementation.

Reply:
Great point. I'm adding this!